proposals/csharp-10.0/parameterless-struct-constructors.md

```diff
@@ -14,6 +14,7 @@ As with [class field initializers](https://github.com/dotnet/csharplang/blob/mas
 
 ### Constructors
 A struct may declare a parameterless instance constructor.
+The struct can be declared as `struct`, `readonly struct`, or `ref struct`.
```

Comment:
Given that we're about to introduce `record struct` perhaps we should flip this to say it's valid for all kinds of `struct` including ... and put the current list.